website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting up Heron with Mesos Cluster Locally on Mac
+---
+
+This is a step by step guide to run Heron on Mesos cluster locally.
+
+## Install Heron
+At the time this article is written, Mesos scheduler is not included in the release version. So you need the latest version of Heron to run it on Mesos. Follow [Compiling](../../../../developers/compiling/compiling) to make sure you can build from the latest version.
+
+## Setting up Apache Mesos Cluster Locally
+
+Follow [Installing Mesos on your Mac with Homebrew](https://mesosphere.com/blog/2014/07/07/installing-mesos-on-your-mac-with-homebrew/) to have Mesos installed and running. To confirm Mesos cluster are ready for accepting Heron topologies, make sure you can access Mesos management console [http://localhost:5050](http://localhost:5050) and see activated slaves.
+
+![console page](/img/mesos-management-console.png)
+
```

Comment:
I didn't actually launch the mesos cluster :), which I don't think it's worth mentioning.